tests/conftest.py

```diff
@@ -144,9 +144,9 @@ async def docker() -> DockerAPI:
 
         docker_images.inspect.return_value = image_inspect
         docker_images.list.return_value = [image_inspect]
-        docker_images.import_image.return_value = [
-            {"stream": "Loaded image: test:latest\n"}
-        ]
+        docker_images.import_image = AsyncMock(
+            return_value=[{"stream": "Loaded image: test:latest\n"}]
+        )
```

Comment:
Ah I see what happened. That method is a `def` in the library not `async def` because it conditionally returns a coroutine based on the `stream` argument. Not conducive to easy mocking like the others, whoops.